### ISSUE TITLE
Abnormal broncho-alveolar lavage fluid morphology

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5543]
**Preferred term label:**
Abnormal broncho-alveolar lavage fluid morphology
**Synonyms**
Abnormality in broncho-alveolar lavage fluid

**Definition (free text, please give PubMed ID)**
see https://pubmed.ncbi.nlm.nih.gov/30173880/

**Parent term (use hpo.jax.org/app)**
Abnormal lung morphology 
https://hpo.jax.org/app/browse/term/HP:0002088

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
-- depends on sub term

**Your nano-attribution (ORCID)**

@mgriese
@GrieseAG

**Comment by pnrobinson** [comment]
Proposed definition
Abnormal type or counts of nucleated immune cells and acellular components in bronchoalveolar lavage (BAL) fluid. BAL is a noninvasive and well-tolerated procedure that is performed with a fiberoptic bronchoscope in the wedged position within a selected bronchopulmonary segment. 

Question: What about red blood cells?

Comment: BAL is commonly used to inform the differential diagnosis of interstitial lung disease or to monitor therapeutic interventions.

**Comment by mgriese** [comment]
BETTER: BAL is a little invasive...(not "noninvasive"; 10% fever afterwards)

Red blood cells are noted (e.g. present in pulmonary hemorrhage syndromes), but not counted, Similarly epithelial cells are only noted, not included numerically into cell differential.

**Comment by pnrobinson** [comment]
adding term as HP:0032973
